include/OpenColorIO/OpenColorTypes.h

```diff
@@ -537,11 +537,18 @@ enum OptimizationFlags : unsigned long
      */
     OPTIMIZATION_LUT_INV_FAST                    = 0x02000000,
 
+    // For CPU processor, in SSE mode, use a faster approximation for log, exp, and pow.
```

Comment:
These new flags should also be added to the Python bindings:
https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/master/src/bindings/python/PyTypes.cpp

Reply:
Added

Reply:
Thanks!